packages/css-library/package.json

```diff
@@ -13,8 +13,7 @@
   "scripts": {
     "build:tokens": "style-dictionary build",
     "build:stylesheets": "sass --load-path=./node_modules/@uswds/uswds/packages/ src/stylesheets:dist/stylesheets",
-    "build:minify": "yarn build:stylesheets --style compressed",
-    "build:minify-core": "sass --style compressed dist/core.css:dist/core.min.css",
+    "build:minify": "yarn run clean-dist && yarn run copy && yarn build:tokens && yarn build:stylesheets --style compressed && yarn copy:scss-files",
```

Comment:
build:minify-core is outdated. The core.css file does not live in that directory